packages/app-store/routing-forms/components/_components/FormSettingsSlideover.tsx

```diff
@@ -0,0 +1,206 @@
+import { Button } from "@calcom/ui/components/button";
+import { Switch, TextAreaField, TextField } from "@calcom/ui/components/form";
+import { Sheet, SheetContent, SheetHeader, SheetTitle, SheetFooter } from "@calcom/ui/components/sheet";
+
+import type { RoutingFormWithResponseCount } from "../../types/types";
+import { TeamMemberSelect } from "./TeamMemberSelect";
+
+type FormSettingsSlideoverProps = {
+  form: RoutingFormWithResponseCount;
+  hookForm: UseFormReturn<RoutingFormWithResponseCount>;
+  isOpen: boolean;
+  onOpenChange: (open: boolean) => void;
+  appUrl: string;
+};
+
+export const FormSettingsSlideover = ({
+  form,
+  hookForm,
+  isOpen,
+  onOpenChange,
+  appUrl,
+}: FormSettingsSlideoverProps) => {
+  const { t } = useLocale();
+  const { data: user } = useMeQuery();
+  const sendUpdatesTo = hookForm.watch("settings.sendUpdatesTo") || [];
+  const sendToAll = hookForm.watch("settings.sendToAll") || false;
+
+  // Store initial form values
+  const initialValuesRef = useRef({
+    name: hookForm.getValues("name"),
+    description: hookForm.getValues("description"),
+    settings: {
+      sendUpdatesTo: hookForm.getValues("settings.sendUpdatesTo") || [],
+      sendToAll: hookForm.getValues("settings.sendToAll") || false,
+      emailOwnerOnSubmission: hookForm.getValues("settings.emailOwnerOnSubmission") || false,
+    },
+  });
+
+  const handleCancel = () => {
+    // Revert only the fields we edit
+    const initialValues = initialValuesRef.current;
+    hookForm.setValue("name", initialValues.name);
+    hookForm.setValue("description", initialValues.description);
+    hookForm.setValue("settings.sendUpdatesTo", initialValues.settings.sendUpdatesTo);
+    hookForm.setValue("settings.sendToAll", initialValues.settings.sendToAll);
+    hookForm.setValue("settings.emailOwnerOnSubmission", initialValues.settings.emailOwnerOnSubmission);
+    onOpenChange(false);
+  };
+
+  return (
+    <Sheet
+      open={isOpen}
+      onOpenChange={(open) => {
+        if (!open) {
+          handleCancel();
+        } else {
+          onOpenChange(open);
+        }
+      }}>
+      <SheetContent className="flex flex-col">
+        <SheetHeader>
+          <SheetTitle>{t("form_settings")}</SheetTitle>
+        </SheetHeader>
+        <div className="mt-6 flex-1 overflow-y-auto">
+          <TextField
+            type="text"
+            containerClassName="mb-6"
+            placeholder={t("title")}
+            {...hookForm.register("name")}
+            data-testid="name"
+          />
+          <TextAreaField
+            rows={3}
+            id="description"
+            data-testid="description"
+            placeholder={t("form_description_placeholder")}
+            {...hookForm.register("description")}
+            defaultValue={form.description || ""}
+          />
+
+          <div className="mt-6">
+            {form.teamId ? (
+              <div className="flex flex-col">
+                <TeamMemberSelect
```

Comment:
It's missing the label 'Send email respones to'
<img width="501" alt="Screenshot 2025-04-30 at 11 03 57 AM" src="https://github.com/user-attachments/assets/890fa36f-3675-400b-a853-2438659bcc02" />